aws-glue/jobs/reporting-ratio_stats.py

```diff
@@ -12,80 +34,69 @@
 job = Job(glueContext)
 
 # Import necessary libraries
```

Comment:
**[Nitpick, non-blocking]** Looks like this comment is no longer in the right place? But I also think it was never really necessary to begin with, we should expect the reader to know what an `import` is for.

```suggestion
```